common/models/transducer/topology.py

```diff
@@ -0,0 +1,44 @@
+from .loss import (rnnt_loss, rnnt_loss_out_type,
+                   rnnt_tf_loss, rna_tf_loss, rna_loss_out_type)
+from .alignment import (rnnt_alignment, rnnt_alignment_out_type,
+                        rna_alignment, rna_alignment_out_type)
+
+
+class Topology:
+  """
+  Class to hold informations about different topologies such as loss-, alignment-funcion and their Data_out_types.
+  loss and alignment functions are to be used in eval like layers that return a source function.
+  """
+  def __init__(self, name: str,
+               loss,
+               loss_out_type,
+               alignment,
+               alignment_out_type):
+    """ Topology such as rnnt, rna, ctc.
+    Args:
+        name (str): name of the alignment
+        loss function (source: (i: int, as_data: bool = False, ...) -> tf.Tensor|Data, ...) -> tf.Tensor
+        loss_out_type function ( (...) -> Data]): loss Data_out_type
+        alignment function (source: (i: int, as_data: bool = False, ...) -> tf.Tensor|Data, ...) -> tf.Tensor
```

Comment:
```suggestion
        alignment: function (source: (i: int, as_data: bool = False, ...) -> tf.Tensor|Data, ...) -> tf.Tensor
```
The interesting things are missing here. What are the (high level) inputs to calculate this alignment, and what do you get out (not just the type, but what is it, what shape?).